vantage/manifest.json

```
@@ -0,0 +1,46 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "f9d211f9-8cd2-4bab-9718-462c5a3fa9ad",
+  "app_id": "vantage",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "This integration allows Datadog users to import their Datadog costs into Vantage.",
```

```suggestion
    "description": "Import your Datadog costs and track them alongside other infrastructure spend",
```

Just a suggestion, but this language is more standard with other integration descriptions.